### PR TITLE
[MC-658] New workflow - Check Host Quarantine Status in FireEye HX from Microsoft Teams

### PR DESCRIPTION
**Pull request by dsliwinski-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

<details>

```
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Executing validator WorkflowEncodingValidator
[*] Executing validator WorkflowParametersKeywordValidator
[*] Executing validator WorkflowUseCaseValidator
[*] Executing validator UnapprovedKeywordsValidator
[*] Executing validator WorkflowPythonScriptUseValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 3152.168ms
```

<summary>
icon-validate .
</summary>
</details>

#### Workflow Builder

workflow

#### Job

check_status_contained
check_status_normal
host_not_found
host_not_provided

### CODE REVIEW

**Review by mgrandys-r7 — DISMISSED** [review]

File: workflows/Check_Host_Quarantine_Status_in_FireEye_HX_from_Microsoft_Teams/help.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+# Documentation
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+This workflow leverages InsightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow.
+
+There are two parameters you will need to configure in order to complete setup of your workflow:
```
Comment (dmandyna-r7) [inline workflows/Check_Host_Quarantine_Status_in_FireEye_HX_from_Microsoft_Teams/help.md:26]:
```suggestion
There are two parameters you will need to be configured in order to complete setup of your workflow:
```

File: workflows/Check_Host_Quarantine_Status_in_FireEye_HX_from_Microsoft_Teams/help.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+* Team Name: The Microsoft Teams team name in your environment where the workflow should be triggered and respond
```
Comment (dmandyna-r7) [inline workflows/Check_Host_Quarantine_Status_in_FireEye_HX_from_Microsoft_Teams/help.md:28]:
```suggestion
* Team Name: The Microsoft Teams team name in your environment where the workflow should be triggered from and where it responds to
```

File: workflows/Check_Host_Quarantine_Status_in_FireEye_HX_from_Microsoft_Teams/help.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* Channel Name: The Microsoft Teams channel name in your environment where the workflow should be triggered and respond (the channel should exist in the aforementioned team)
```
Comment (dmandyna-r7) [inline workflows/Check_Host_Quarantine_Status_in_FireEye_HX_from_Microsoft_Teams/help.md:29]:
```suggestion
* Channel Name: The Microsoft Teams channel name in your environment where the workflow should be triggered from and where responds to (the channel should exist in the aforementioned team)
```

**Review by dmandyna-r7 — CHANGES_REQUESTED** [review]
Looks good after fixing the wording in help.md.

**Review by mgrandys-r7 — APPROVED** [review]

**Pull request merged**